src/Microsoft.AspNet.Mvc.Core/ActionResults/ContentResult.cs

```diff
@@ -26,17 +34,26 @@ public class ContentResult : ActionResult
         {
             var response = context.HttpContext.Response;
 
-            MediaTypeHeaderValue contentTypeHeader;
-            if (string.IsNullOrEmpty(ContentType))
+            var contentTypeHeader = ContentType;
+            Encoding encoding;
+            if (contentTypeHeader == null)
             {
-                contentTypeHeader = new MediaTypeHeaderValue("text/plain");
+                contentTypeHeader = DefaultContentType;
+                encoding = DefaultContentType.Encoding;
```

Comment:
We should use the correct encoding (without BOM) even if it doesn't have an impact. 

This breaks the cycle of people have to ask why we're using plain UTF8 and you having to explain it again.